scripts/lib/install-wal-g

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -eux
+
+apt-get install -y \
+    liblzo2-dev \
+    libbrotli-dev \
+    libsodium-dev \
+    build-essential \
+    gcc \
+    cmake \
+    libc-dev
```

Comment:
I think this script should probably be called `build-wal-g`, since that's our naming convention from `scripts/lib/build-pgroonga`.

Reply:
I mean, we also have `scripts/lib/install-node` and `scripts/lib/install-yarn` but I agree it's a little more like pgroonga.

Reply:
Yeah, the naming convention to me is about whether it actually builds something or not.